### ISSUE TITLE
[bug] <pf-accordion> Accessibility: arrow keys inside nested accordions don't work as expected

### ISSUE THREAD

**Issue by markcaron** [patternfly/patternfly-elements#2490]
When navigating an accordion inside another accordion, the arrow keys don't move through the nested accordion. Instead, they jump up to the parent accordion.

![image](https://user-images.githubusercontent.com/1890950/235973874-05272abe-aac7-48c1-93bd-aa5e7c29fbf5.png)

**Steps to reproduce:**

1. Run accordion demo locally
2. Find nested example
3. Expand first accordion item
4. Tab inside to the child accordion
5. Hit the down arrow (assuming it should go to the next item in the child accordion)
6. Focus jumps to the parent accordion header

**Comment by bennypowers** [comment]
@nikkimk what's the expected behaviour here? wai-aria guidelines say that UP/down switch between headers, but don't mention switching between headers of separate instances of accordion

**Comment by nikkimk** [comment]
@bennypowers arrow keys should only work on one level of accordion at a time. There would be a tab to the panel where the accordion is nested. Once the nested accordion has focus, then it would be arrow keys. (Jump links is different from accordion because the nesting occurs within the same parent structure, there is no tab key to get to the children.)

**Comment by bennypowers** [comment]
should have been fixed by #2493
